.github/workflows/build.yml

```diff
@@ -469,12 +469,13 @@ jobs:
           REDHAT_TO_COMMUNITY: ${{ needs.chart-verifier.outputs.redhat_to_community }}
           WEB_CATALOG_ONLY: ${{ needs.chart-verifier.outputs.web_catalog_only }}
           OCP_VERSION_RANGE: ${{ steps.get-ocp-range.outputs.ocp-version-range }}
+          API_URL: ${{ github.event.pull_request._links.self.href }}
         id: prepare-chart-release
         run: |
           cd pr-branch
           ../ve1/bin/chart-repo-manager \
-            --repository=${{ github.repository }} \
-            --api-url=${{ github.event.pull_request._links.self.href }} \
+            --repository="$GITHUB_REPOSITORY" \
```

Comment:
Initially, I think I intended for this to be `${{ env.GITHUB_REPOSITORY }}`, but I think what you've done here makes sense, given the original goal I had in mind.

Looking at this now, I wonder if there are any downsides to using shell-specific environment variables here. I can't think of any - is there anything you can think of?